products/BellHybrid/apps/application-bell-powernap/windows/PowerNapProgressWindow.cpp

```diff
@@ -10,16 +10,16 @@
 
 namespace
 {
-    inline constexpr auto powernapTimerName = "PowerNapTimer";
-    inline constexpr std::chrono::seconds timerTick{1};
+    constexpr auto powerNapTimerName = "PowerNapTimer";
+    constexpr std::chrono::seconds powerNapTimerPeriod{1};
```

Comment:
What about `std::chrono::seconds{1};` and use `auto` keyword?

Reply:
Done